### PR TITLE
[Events] Fix issues on anomalies filters and on anomalies snoozing

### PR DESCRIPTION
**Pull request by jgchoppe**
This PR is about the issue we had on anomalies filters and anomalies snoozing (the fact that we couldn't add or delete filters and we couldn't snooze or unsnooze anomalies).

### CODE REVIEW

File: costs/anomalies/anomalies_route.go [hunk 1]
```diff
@@ -19,6 +19,7 @@ import (
 	"database/sql"
 	"encoding/json"
 	"fmt"
+	"github.com/trackit/trackit/cache"
```
Comment (hug33k) [inline costs/anomalies/anomalies_route.go:22]: Wrong import order

File: costs/anomalies/anomalies_route.go [hunk 2]
```diff
@@ -69,11 +69,11 @@ func init() {
 			db.RequestTransaction{Db: db.Db},
 			users.RequireAuthenticatedUser{users.ViewerAsParent},
 			routes.QueryArgs(anomalyQueryArgs),
-			cache.UsersCache{},
 			routes.Documentation{
 				Summary:     "get the cost anomalies",
 				Description: "Responds with the cost anomalies based on the query args passed to it",
 			},
+			cache.UsersCache{},
```
Comment (hug33k) [inline costs/anomalies/anomalies_route.go:76]: Why did you moved this line ?

File: costs/anomalies/filters_route.go [hunk 3]
```diff
@@ -4,11 +4,11 @@ import (
 	"database/sql"
 	"encoding/json"
 	"errors"
-	"net/http"
-
 	"github.com/trackit/jsonlog"
-
+	"github.com/trackit/trackit/aws"
 	"github.com/trackit/trackit/cache"
+	"net/http"
```
Comment (hug33k) [inline costs/anomalies/filters_route.go:10]: Wrong import order

File: costs/anomalies/filters_route.go [hunk 4]
```diff
@@ -146,4 +157,4 @@ func postAnomaliesFiltersWithValidFilters(r *http.Request, tx *sql.Tx, dbUser *m
 		}
 	}
 	return http.StatusInternalServerError, errors.New("Failed to update filters.")
-}
+}
```
Comment (hug33k) [inline costs/anomalies/filters_route.go:160]: Missing new line

File: costs/anomalies/snoozing_route.go [hunk 5]
```diff
@@ -16,13 +16,14 @@ package anomalies
...
-	"net/http"
-
+	"github.com/trackit/jsonlog"
+	"github.com/trackit/trackit/aws"
 	"github.com/trackit/trackit/cache"
 	"github.com/trackit/trackit/db"
 	"github.com/trackit/trackit/models"
 	"github.com/trackit/trackit/routes"
 	"github.com/trackit/trackit/users"
+	"net/http"
```
Comment (hug33k) [inline costs/anomalies/snoozing_route.go:26]: Wrong import order

**Review by hug33k — APPROVED** [review]

**Review by hug33k — CHANGES_REQUESTED** [review]
See comments

**Comment by Dr0p42** [comment]
@jgchoppe I don't know which IDE/Text editor you are using, but most of them have plugins that can run `gofmt` and `goimports` fo you when saving your files ;)

**Pull request closed without merging**